pkg/webhook/resources/restore/validator.go

```diff
@@ -92,6 +96,12 @@ func (v *restoreValidator) Create(request *types.Request, newObj runtime.Object)
 		return werror.NewInvalidError(fmt.Sprintf("please stop the VM %q before doing a restore", vm.Name), fieldTargetName)
 	}
 
+	if result, err := util.HasInProgressingVMRestoreOnSameTarget(v.vmRestore, vm.Namespace, vm.Name); err != nil {
+		return werror.NewInternalError(fmt.Sprintf("failed to get the VM-related restores, err: %+v", err))
```

Comment:
I've found that we use uppercase in some places and lowercase in others; it might be better to be consistent with using uppercase only.

Reply:
Update it. Thank you.